osu.Game/Rulesets/UI/Playfield.cs

```diff
@@ -51,6 +53,10 @@ public abstract class Playfield : CompositeDrawable
         /// </summary>
         public readonly BindableBool DisplayJudgements = new BindableBool(true);
 
+        private Bindable<bool> showPlayfieldArea;
+        private Bindable<double> playfieldAreaDimLevel;
+        private EditorPlayfieldBorder playfieldArea;
```

Comment:
Gameplay elements should not be using editor-specific components. If this is to be reused then `EditorPlayfieldBorder` should be renamed/moved to reflect its more general purpose.